src/test/denialofservice_tests.cpp

```diff
@@ -45,10 +45,19 @@ BOOST_FIXTURE_TEST_SUITE(denialofservice_tests, TestingSetup)
 // work.
 BOOST_AUTO_TEST_CASE(outbound_slow_chain_eviction)
 {
-    ConnmanTestMsg& connman = static_cast<ConnmanTestMsg&>(*m_node.connman);
-    // Disable inactivity checks for this test to avoid interference
-    connman.SetPeerConnectTimeout(99999s);
-    PeerManager& peerman = *m_node.peerman;
+    auto evictionman = std::make_unique<EvictionManager>();
```

Comment:
Why is this switched away from using `m_node`? Not that I object, it seems reasonable to me.

If it's just because "it's the only use of `m_node.connman` in these tests", maybe break that change out as a separate commit?